### ISSUE TITLE
Breadcrumbs (Links) and Buttons fail WCAG 2 AA Color Contrast

### ISSUE THREAD

**Issue by ConanSage** [Sage/carbon#4417]
### Current behaviour

Breadcrumbs (links) and Buttons do not conform to WCAG 2 AA contrast ratio thresholds.

The expected contrast ratio must be at a minimum equal to:
4.5:1

Button Carbon element:
https://carbon.sage.com/?path=/docs/button--primary

Link Carbon element:
https://carbon.sage.com/?path=/docs/link--default-story

### Expected behaviour

Breadcrumbs and Buttons conform to WCAG 2 AA contrast ratio thresholds.

The expected contrast ratio must be at a minimum equal to:
4.5:1

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/docs/button--primary

### JIRA Ticket (Sage Only)

SBS-20736

### Suggested Solution

Bump the Color contrast so it passes WCAG 2 AA contrast ratio thresholds

### Carbon Version

84.1.0

### What browsers are you seeing the problem on?

Chrome

### What Operating System are you seeing the problem on?

Windows

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by sjohnson-sage** [comment]
Hi @ConanSage what tool are you using to check this colour contrast? We have a tool checking colour contrasts which says it matches WCAG 2 AA guidlines

**Comment by ConanSage** [comment]
I used AXE, and also WCAG - Contrast checker - both are Chrome extensions.

**Comment by sjohnson-sage** [comment]
Thanks @ConanSage - I believe this isn't picked up by our tests as we only currently have a white background on our tertiary button stories etc 

I think we need UX input because we just have a transparent background for our tertiary buttons and links, so if we change the colour of the text it doesn't necessarily solve the underlying issue here.

@harpalsingh this is similar to https://github.com/Sage/carbon/issues/4384 - where the contrast doesn't meet WCAG 2 AA standards when over a grey background - should we be specifying what background colours to use behind our links and buttons? Thanks

**Comment by ZhuoyuJin** [comment]
Similar accessibility issue happens to Pod component, 'soft-delete-state', and 'address-example'.

**Comment by hegou33** [comment]
Similar accessibility issue happens to label of Textbox/Select/MultiSelect component in Dialog.

**Comment by clairedenning** [comment]
I would like to help but none of the links work for me. What colours are your buttons, links and backgrounds?

**Comment by harpalsingh** [comment]
@clairedenning This is resolved when the new colors are applied, to fix this issue now we would need to really apply a full update to the base Mint color so it works against the app background, or (much harder task) is updated each component where the text can appear on a app background and updated, so this is a known issue and resolved with all the new brand work. so does this need anything done if those updates are imminent and the Mint is only live in the initial release of CM.

**Comment by ConanSage** [comment]
I have just updated the links for this issue

**Comment by ConanSage** [comment]
> I would like to help but none of the links work for me. What colours are your buttons, links and backgrounds?

I have just updated the links for this issue

**Comment by ConanSage** [comment]
> @clairedenning This is resolved when the new colors are applied, to fix this issue now we would need to really apply a full update to the base Mint color so it works against the app background, or (much harder task) is updated each component where the text can appear on a app background and updated, so this is a known issue and resolved with all the new brand work. so does this need anything done if those updates are imminent and the Mint is only live in the initial release of CM.

Thank you for the update on this. Perhaps we should leave this issue open until we can confirm no WCAG Color Contrast issues are present on the new brand work?

**Comment by DlgSHi** [comment]
@ConanSage thank you for this, we are closing this issue because as @harpalsingh said we will update the `Mint` theme in a nearest future.
